juniper/Cargo.toml

```diff
@@ -54,7 +54,7 @@ uuid = { version = "0.8", default-features = false, optional = true }
 bencher = "0.1.2"
 pretty_assertions = "0.6.1"
 serde_json = "1.0.2"
-tokio = { version = "0.2", features = ["macros", "rt-core", "time"] }
+tokio = { version = "1.0.2", features = ["macros", "rt-multi-thread", "time"] }
```

Comment:
Is there a reason to pin to 1.0.2 instead of the latest 1.2.0?

Reply:
> Is there a reason to pin to 1.0.2 instead of the latest 1.2.0?

This means "1.x.x >= 1.0.2", i.e. "^1.0.2", not "=1.0.2". It's best for libraries to refer to the lowest version required that is semver-compatible with latest, in case a host project needs to use an earlier, but still compatible version.